modules/openxBidAdapter.md

```diff
@@ -15,14 +15,15 @@ Publishers are welcome to test the other adapter and give feedback. Please note
+| Name                            | Scope | Type | Description                                                                                                                                                                                                                                                                                                                                                                                                                        | Example
+|---------------------------------| ----- | ---- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------
+| `delDomain` ~~or `platform`~~** | required | String | OpenX delivery domain provided by your OpenX representative.                                                                                                                                                                                                                                                                                                                                                                       | "PUBLISHER-d.openx.net"
+| `unit`                          | required | String | OpenX ad unit ID provided by your OpenX representative.                                                                                                                                                                                                                                                                                                                                                                            | "1611023122"
+| `customParams`                  | optional | Object | User-defined targeting key-value pairs. customParams applies to a specific unit.                                                                                                                                                                                                                                                                                                                                                   | `{key1: "v1", key2: ["v2","v3"]}`
+| `customFloor`                   | optional | Number | Minimum price in USD. customFloor applies to a specific unit. For example, use the following value to set a $1.50 floor: 1.50 <br/><br/> **WARNING:**<br/> Misuse of this parameter can impact revenue<br/><br/>Note: OpenX suggests using the [Price Floor Module](https://docs.prebid.org/dev-docs/modules/floors.html) instead of customFloor. The Price Floor Module will be prioritized over customFloor if both are present. | 1.50
```

Comment:
`will be prioritized` is it true now or in the future ?
if true now, `is prioritized` instead of `will be prioritized` seems less ambiguous

Reply:
good point, it is already happening, since PR was merged in an upstream I will keep in mind to change it next time